src/components/GuestForm/GuestForm.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { useEntries } from '../../context/EntryContext';
+import { useUser } from '../../context/UserContext';
+
+export default function GuestForm() {
+  const { user, setUser } = useUser();
+  const { setEntries } = useEntries();
+
+  const [name, setName] = useState('');
+  const [newEntry, setNewEntry] = useState('');
+
+  function updateEntryList() {
+    setUser(name);
+    setEntries((prevState) => [...prevState, { name: name, entry: newEntry }]);
+    setNewEntry('');
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    updateEntryList();
+  };
+
+  const handleName = (e) => {
+    e.preventDefault();
```

Comment:
Any type of event, you can cancel the defaultBehavior, but in this instance it should be fine without it. Mainly we use it for form submissions.